<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmconnect</class>
 <widget class="QDialog" name="frmconnect">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>394</width>
    <height>230</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Connect to Database</string>
  </property>
  <property name="windowIcon">
   <iconset resource="app_new.qrc">
    <normaloff>:/app_new/medfisis.ico</normaloff>:/app_new/medfisis.ico</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="lbHost">
     <property name="text">
      <string>Host</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>lineHost</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="lineHost"/>
   </item>
   <item row="0" column="2" rowspan="3">
    <layout class="QVBoxLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="okButton">
       <property name="text">
        <string>OK</string>
       </property>
       <property name="icon">
        <iconset resource="app_new.qrc">
         <normaloff>:/app_new/ok.png</normaloff>:/app_new/ok.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
       <property name="icon">
        <iconset resource="app_new.qrc">
         <normaloff>:/app_new/cancel.png</normaloff>:/app_new/cancel.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="lbDataSource">
     <property name="text">
      <string>Data Source</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>lineDataSource</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="lineDataSource"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="lbUsername">
     <property name="text">
      <string>Username</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>lineUsername</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="lineUsername"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="lbPassword">
     <property name="text">
      <string>Password</string>
     </property>
     <property name="buddy">
      <cstring>linePassword</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="linePassword">
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="lbAlias">
     <property name="text">
      <string>Alias</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>lineAlias</cstring>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLineEdit" name="lineAlias"/>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="lbDriver">
     <property name="text">
      <string>Driver</string>
     </property>
     <property name="buddy">
      <cstring>cmbDriver</cstring>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QComboBox" name="cmbDriver"/>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lineHost</tabstop>
  <tabstop>lineDataSource</tabstop>
  <tabstop>lineUsername</tabstop>
  <tabstop>linePassword</tabstop>
  <tabstop>lineAlias</tabstop>
  <tabstop>cmbDriver</tabstop>
  <tabstop>okButton</tabstop>
  <tabstop>cancelButton</tabstop>
 </tabstops>
 <resources>
  <include location="app_new.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>okButton</sender>
   <signal>clicked()</signal>
   <receiver>frmconnect</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>351</x>
     <y>38</y>
    </hint>
    <hint type="destinationlabel">
     <x>233</x>
     <y>29</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>frmconnect</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>330</x>
     <y>66</y>
    </hint>
    <hint type="destinationlabel">
     <x>197</x>
     <y>125</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
